python_bindings/tests/bindings_test_labels.py

```diff
@@ -1,131 +1,127 @@
 import os
 import unittest
 
+import numpy as np
 
-class RandomSelfTestCase(unittest.TestCase):
-    def testRandomSelf(self):
-      for idx in range(16):
```

Comment:
Fixed inconsistent indentation. It was a mix of 2 and 4 spaces in this file. Now it is 4 spaces.